flang/lib/Optimizer/Transforms/DebugTypeGenerator.cpp

```diff
@@ -229,6 +253,9 @@ DebugTypeGenerator::convertType(mlir::Type Ty, mlir::LLVM::DIFileAttr fileAttr,
     if (auto seqTy = mlir::dyn_cast_or_null<fir::SequenceType>(elTy))
       return convertBoxedSequenceType(seqTy, fileAttr, scope, loc, false,
                                       false);
+    if (auto heapTy = mlir::dyn_cast_or_null<fir::HeapType>(elTy))
+      return convertPointerLikeType(heapTy.getElementType(), fileAttr, scope,
+                                    loc, true, false);
```

Comment:
nit: please add comments with argument names for the literal arguments.

Reply:
I will fix this and will also open a separate PR to cleanup the comments throughout the file.